src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,69 @@
+import classNames from 'classnames';
+import { Person } from '../types';
+import { PersonLink } from './PersonLink';
+
+type Props = {
+  people: Person[];
+  slug: string;
+};
+
+export const PeopleTable: React.FC<Props> = ({ people, slug }) => {
+  const parentsHandler = (parentName: string | null) => {
+    const parent = people.find(person => person.name === parentName);
+
+    if (parentName && parent) {
+      return (
+        <PersonLink person={parent} />
+      );
+    }
+
+    if (parentName && !parent) {
+      return parentName;
+    }
```

Comment:
you can move those blocks under one if statement 
```suggestion
    if (parentName) {
      if (parent) {
        return (
          <PersonLink person={parent} />
        );
      }
      
      return parentName;
    }
```